rust/pypasswords/src/lib.rs

```
@@ -5,58 +5,245 @@
     }
 
     #[cfg(feature = "cbc")]
     #[pyfunction]
     /// Encrypt the data with CBC `TripleDES`
     pub(crate) fn cbc_encrypt(password: &str, data: &str) -> PyResult<String> {
         let result_bytes = passwords::cbc_encrypt(password.as_bytes(), data.as_bytes())
-            .map_err(|err| PyRuntimeError::new_err(err.to_string()))?;
-        String::from_utf8(result_bytes)
-            .map_err(|_err| PyRuntimeError::new_err("Base64 output contained invalid UTF-8"))
+            .map_err(ToPythonError::to_python_error)?;
+        Ok(String::from_utf8(result_bytes).expect("Base64 output should only contain ASCII"))
```

Clippy (1.96.0)

[error] used `expect()` on a `Result` value

(error)